python3: can't open file 'metrics_consumer.py': [Errno 2] No such file or directory
python3: can't open file 'metrics_consumer.py': [Errno 2] No such file or directory
